pages/pil/create/content/index.js

```diff
@@ -7,9 +7,6 @@ module.exports = {
   catE: {
     title: 'Category E',
     subtitle:
-      'These personal licenses are issued for short-term educational and training purposes. They will only authorise you to work on a specific project.'
-  },
-  buttons: {
-    apply: 'Apply now'
```

Comment:
Is there a reason "Apply now" is removed?

Reply:
Because the application process for category E PILs is not a part of our system.

To apply a category E PIL, applicants should contact their establishment’s SPoC for advice or email ASPA London (as per the other change).

Reply:
Context: It shouldn't be removed because it's used for the button under Cat ABCDF PILs.